openmls/src/group/mls_group/mod.rs

```diff
@@ -549,9 +1121,29 @@ impl MlsGroup {
 pub struct StagedWelcome {
     // The group configuration. See [`MlsGroupJoinConfig`] for more information.
     mls_group_config: MlsGroupJoinConfig,
-    // The internal `CoreGroup` used for lower level operations. See `CoreGroup` for more
-    // information.
-    group: StagedCoreWelcome,
+    public_group: PublicGroup,
+    group_epoch_secrets: GroupEpochSecrets,
+    own_leaf_index: LeafNodeIndex,
+
+    /// A [`MessageSecretsStore`] that stores message secrets.
+    /// By default this store has the length of 1, i.e. only the [`MessageSecrets`]
+    /// of the current epoch is kept.
+    /// If more secrets from past epochs should be kept in order to be
+    /// able to decrypt application messages from previous epochs, the size of
+    /// the store must be increased through [`max_past_epochs()`].
+    message_secrets_store: MessageSecretsStore,
+
+    /// Resumption psk store. This is where the resumption psks are kept in a rollover list.
+    pub(crate) resumption_psk_store: ResumptionPskStore,
```

Comment:
Not necessarily a big issue, but I don't think we mix fields with different visibility in the library. I think we usually have getters instead, unless all fields are `pub`.

Reply:
All I did was inline this from `StagedCoreWelcome`. It was public there, too. I'd rather pull this straight in a follow-up.

Reply:
This is the only outstanding issue, why not fix it in this PR, like the rest of it?